src/dash-table/derived/cell/eventHandlerProps.ts

```diff
@@ -74,6 +85,13 @@ class EventHandler {
                         rowIndex,
                         columnIndex
                     );
+                case Handler.MoveHeader:
+                    return handleMoveHeader.bind(
+                        undefined,
+                        this.propsFn,
+                        rowIndex,
+                        columnIndex
+                    );
```

Comment:
New event handlers for header cells, equivalent but slightly different from the same handlers for the data cells